odoo/addons/base/models/ir_actions_report.py

```diff
@@ -804,6 +804,9 @@ def _render_qweb_pdf(self, report_ref, res_ids=None, data=None):
                     continue
 
                 record = self.env[report_sudo.model].browse(res_id)
+                # unable to fetch the record, it won't be saved as we can't split the documents unambiguously
```

Comment:
"unable to fetch the record" is not very clear since we are not unable to fetch the record, we just have no record to fetch

we also could do the test above .browse with res_id

Reply:
I think current code makes sense: we still need the stream with False key that contain the document that could not be split.

But reading the code I see another issue: if we print document 1, 2, 3, 4 and 2 and 4 are already saved attachment (eg. they were printed one by one before the multi print). If we have an error when splitting document 1 and 3, we return:

```py
                    return {False: {
                        'report_action': self,
                        'stream': pdf_content_stream,
                        'attachment': None,
                    }}
```
that only contain document 1 and 3, instead of also returning the stream of document 2 and 4.